bleep-cli/src/scala/bleep/Main.scala

```diff
@@ -122,6 +122,12 @@ object Main {
 
     val watch = Opts.flag("watch", "start in watch mode", "w").orFalse
 
+    val updateAsScalaSteward = Opts.flag("steward", "Use same upgrade strategy as Scala Steward").orFalse
```

Comment:
are you able to briefly explain what the strategy is? bump minor to newest, and if minor is newest then pick newest major?